File: Cryptography and security libraries and algorithms/Python/Tabnine/Hash generation.py
import hashlib
import scrypt
def Hash_Pass_1(password):
    NewHash=hashlib.md5(password)
    return(NewHash)
def Hash_Pass_2(password):
    NewHash=scrypt.hash(password,salt=None,n=2**14,r=8,p=1)
    return(NewHash)
MyPassword="123Pass"
#generate the hash for MyPassword and store it in a variable called MyHash
MyHash=Hash_Pass_1(MyPassword)
print(MyHash)
print(Hash_Pass_2(MyPassword))

File: Cryptography and security libraries and algorithms/Javascript/Tabnine/Hash generation.js
const crypto = require('crypto');

function Hash_Pass_1(password) {
    const hash = crypto.createHash('md5').update(password).digest('hex');
    return hash;
}

function Hash_Pass_2(password) {
    const hash = crypto.scryptSync(password, null, 64, { N: 1<<14, r: 8, p: 1 });
    return hash.toString('hex');
}

const password = "123Pass";

// generate the hash for MyPassword and store it in a variable called MyHash
const MyHash = Hash_Pass_1(password);
console.log(MyHash);